File: add/codesnippet/CPP/p-system.net.sockets.tcp_11_1.cpp
      // Sets the receive time out using the ReceiveTimeout public property.
      tcpClient->ReceiveTimeout = 5;
      
      // Gets the receive time out using the ReceiveTimeout public property.
      if ( tcpClient->ReceiveTimeout == 5 )
            Console::WriteLine( "The receive time out limit was successfully set {0}", tcpClient->ReceiveTimeout );

      

File: add/codesnippet/CSharp/p-system.net.sockets.tcp_11_1.cs
        // Sets the receive time out using the ReceiveTimeout public property.
        tcpClient.ReceiveTimeout = 5000;

        // Gets the receive time out using the ReceiveTimeout public property.
        if (tcpClient.ReceiveTimeout == 5000)
            Console.WriteLine ("The receive time out limit was successfully set " + tcpClient.ReceiveTimeout.ToString ());
